CONTRIBUTING.md

```diff
@@ -103,10 +103,12 @@ python -m venv venv
 source venv/bin/activate
 # update the version of pip, setuptools, and wheel
 (venv) pip install -U pip setuptools wheel
-# Install the dependencies in your virtual environment
-(venv) pip install .[all]
+# Install all the dependencies in your virtual environment.
+# The dot or period at the end means that the dependencies are referenced is in the same directory as the current location).
+(venv) pip install .
```

Comment:
leaving out the `[all]` means that you will miss dependencies. Perhaps `zsh` requires additional quoting that `bash` does not: `pip install ".[all]"`?

@kinow Since we don't currently plan to publish your forked `sphinx/run_cmd.py`, why not move all the extra dependencies to be part of the main `install_requires`? When do we want to only install some of these?

Reply:
Oh, I did not know that. Thank you so much. 
Will do that @mr-c

Reply:
Can you have a kindly look @mr-c ?